File: exercises/beecrowd/1066/1066.go
// https://judge.beecrowd.com/pt/problems/view/1066
package main

import (
	"fmt"
)

func main() {
	var v1, v2, v3, v4, v5 int

	fmt.Scanf("%d", &v1)
	fmt.Scanf("%d", &v2)
	fmt.Scanf("%d", &v3)
	fmt.Scanf("%d", &v4)
	fmt.Scanf("%d", &v5)

	values := []int{v1, v2, v3, v4, v5}
	var countEven, countOdd, countPositive, countNegative int

	for _, v := range values {
		if v%2 == 0 {
			countEven++
		} else {
			countOdd++
		}

		if v > 0 {
			countPositive++
		} else if v < 0 {
			countNegative++
		}
	}

	fmt.Printf("%d valor(es) par(es)\n", countEven)
	fmt.Printf("%d valor(es) impar(es)\n", countOdd)
	fmt.Printf("%d valor(es) positivo(s)\n", countPositive)
	fmt.Printf("%d valor(es) negativo(s)\n", countNegative)
}

File: exercises/beecrowd/1066/1066.java
// https://judge.beecrowd.com/pt/problems/view/1066
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int pares = 0, impares = 0, pos = 0, neg = 0;

        for (int i = 0; i < 5; i++) {
            int v = sc.nextInt();
            if (v % 2 == 0) pares++;
            else impares++;
            if (v > 0) pos++;
            else if (v < 0) neg++;
        }

        System.out.println(pares + " valor(es) par(es)");
        System.out.println(impares + " valor(es) impar(es)");
        System.out.println(pos + " valor(es) positivo(s)");
        System.out.println(neg + " valor(es) negativo(s)");
    }
}
